todo_app/data/trello_items.py

```diff
@@ -0,0 +1,114 @@
+import requests
+import os
+
+
+# board_id = os.getenv(TRELLO_BOARD_ID)
+# trello_token = os.getenv(TRELLO_TOKEN)
+# trello_key = os.getenv(TRELLO_KEY)
+
+class Trello:
+    def __init__(self):
+        self.board_id = os.getenv('TRELLO_BOARD_ID')
+        self.trello_token = os.getenv('TRELLO_TOKEN')
+        self.trello_key = os.getenv('TRELLO_KEY')
+        self.lists = self.get_lists()
+
+    def get_lists(self):
+        """
+        Gets the list names and list ids
+
+        Returns:
+            dict: The dictionary with list ids and their retrospective names
+
+        """
+        trello_lists_api = requests.get(
+            f"https://api.trello.com/1/boards/{self.board_id}/lists?key={self.trello_key}&token={self.trello_token}").json()
+        list_names = {}
+        for list_name in trello_lists_api:
+            list_names[list_name['id']] = list_name['name']
+        return list_names
+
+
+class Item(Trello):
+    def __init__(self, item_id, name, status):
+        super().__init__()
```

Comment:
Why is this class inheriting from `Trello`, what behaviour are you hoping to capture from the parent class?

I'd argue that an individual Item is fundamentally quite different from the wider `Trello` service (and arguably one of the purposes of the `Item` class is to add a separation between our internal concept of to-dos, and the fact we're storing them in Trello)

Possibly stronger yet though, is to see the effect on performance this is having:
* Try loading your homepage with half a dozen Items showing
* Then remove the Trello inheritance as below, and try again:
```suggestion
class Item():
    def __init__(self, item_id, name, status):
```

Hopefully you'll see what I do, which is a massive improvement in the performance. The reason for that, is that currently every single `Item` that you create runs the Trello `__init__` function, which makes a request to load our Trello lists. Which means if I have 7 items on the page, then I'm making 7 additional HTTP requests, and that quickly adds up how long the page takes to load!

Reply:
I have made the suggested changes and I can see an improvement in the performance